validation/info/fields_validators.go

```diff
@@ -51,11 +51,16 @@ func ValidateAssetRequiredKeys(a AssetModel) error {
 }
 
 func ValidateAssetType(assetType string, chain coin.Coin) error {
-	chainFromType, err := types.GetChainFromAssetType(assetType)
+	chainFromTypeRaw, err := types.GetChainFromAssetType(assetType)
 	if err != nil {
 		return fmt.Errorf("failed to get chain from asset type: %w", err)
 	}
 
+	chainFromType := chainFromTypeRaw
+	if chainFromTypeRaw.ID == coin.CRONOS {
+		chainFromType = coin.Cryptoorg()
```

Comment:
can we update this information on `assets` repo?

Reply:
unfortunately we can't without copypasting whole validate func from here

Reply:
wait, I haven't approved this PR.

I saw this PR https://github.com/trustwallet/assets/pull/25575/files is fixing the wrong directory, so why do we still need this change?

Reply:
In `check` CI action we had 2 type of errors:
1. Invalid explorer url. It was fixed by https://github.com/trustwallet/assets/pull/25575/files
2. When trying to get corresponding chain to CRC20 token, it returns Cronos which causes other type of validation error. It was fixed by this line

Reply:
Got it, thank you.
May I ask you to add comments above your changes?

Reply:
Sure, added https://github.com/trustwallet/assets-go-libs/pull/103